README.md

```diff
@@ -1,3 +1,8 @@
+# Rules of engagement
```

Comment:
It might read better if we put this section below "Project suggestions" instead of above